src/components/profile/ProfileService.js

```diff
@@ -331,9 +361,9 @@ goog.require('ga_urlutils_service');
 
           //Icon for elevation up
           group.append('text')
-              .attr('class', 'ga-profile-icon')
+              .attr('font-family', 'FontAwesome')
```

Comment:
I guess it's because it's not a `ga-profile-icon` :)

Use a `ga-profile-text`class to avoid duplicate style definition